src/UI/Command.php

```diff
@@ -101,6 +101,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->printHeaderOnce($output);
             throw $e;
         }
+
+        return 0;
```

Comment:
Is it necessary?

Reply:
Yep! The new version of symfony command require an exit code!

Reply:
Well, we should add return type hint in this case or is not necessary?

Reply:
It’s a bullet of the roadmap! ;)